cogs/events.py

```diff
@@ -32,11 +33,66 @@ async def on_error(self, event, *args, **kwargs):
             title=event
         )
 
-        if not self.dev_mode:
-            await self.error_hook.send(embed=em)
+        if not constants.DEV_MODE:
+            # await self.error_hook.send(embed=em)
+            await self.client.send_message(embed=em)
         else:
             traceback.print_exc()
 
+    @commands.Cog.listener()
+    async def on_guild_join(self, guild: discord.Guild):
+
+        hw = self.client.get_user(self.client.user.id)
```

Comment:
Since `self.client.user` will return `discord.User` instance, there is no need to get ID from that and get the user from it using `self.client.get_user`. We can simply do:
```suggestion
        hw = self.client.user
```